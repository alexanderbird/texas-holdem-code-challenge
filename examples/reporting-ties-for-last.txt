2C 4C 6C 8S TS
Lexine 3D JS
Laney 3C KS
Francois 3H JH
Evelin 3S AH
