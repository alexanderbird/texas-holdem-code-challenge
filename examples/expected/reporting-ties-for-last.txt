1 Evelin Highcard Ace
2 Laney Highcard King
4 Lexine Highcard Jack
4 Francois Highcard Jack
